### PR TITLE
Set kernel.softlockup_panic =1 based on the flag.

### PR DESCRIPTION
**Pull request by dchen1107**
ref: https://github.com/kubernetes/kubernetes/issues/37853

### CODE REVIEW

**Comment by dchen1107** [comment]
I created the cluster and checked the configs on the node:

# cat /proc/sys/kernel/softlockup_panic
1
# cat /proc/sys/kernel/softlockup_all_cpu_backtrace 
1

**Comment by dchen1107** [comment]
cc/ @saad-ali

File: cluster/saltbase/salt/docker/init.sls [hunk 1]
```diff
@@ -261,6 +261,20 @@ net.ipv4.ip_forward:
   sysctl.present:
     - value: 1
 
+{% if pillar.get('softlockup_panic', '').lower() == 'true' %}
```
Comment (vishh) [inline cluster/saltbase/salt/docker/init.sls]: nit: Why add this to `docker` salt scripts?

Comment (dchen1107) [inline cluster/saltbase/salt/docker/init.sls]: Another place is kubelet salt scripts. It doesn't matter at kubelet or docker salt script, it just need to run on every CVM node.

**Pull request merged**